packages/client/mutations/DowngradeToStarterMutation.ts

```diff
@@ -8,6 +8,10 @@ graphql`
     organization {
       tier
       billingTier
+      viewerTeams {
```

Comment:
Without this, tier & isPaid aren't updated in the teams and the user needs to refresh the page to see the updated settings. This mirrors the upgrade mutation query.